diagnostic.md

```diff
@@ -21,13 +21,22 @@ Write the `curl` request you'd use to retrieve a list of all donuts on the
 server.
 
 ```sh
-# your answer here
+curl -- request GET http://www.example.com/donuts \
```

Comment:
Careful, that should be `--request` (no space).

Also, you need `--include` so you can display headers.
> "You are expected to use the appropriate HTTP method, and to ask curl to include HTTP headers in its output."